en/docs/how-tos/security/api-authentication/mutual-ssl-authentication.md

```diff
@@ -17,6 +17,41 @@ x-wso2-mutual-ssl: "mandatory" # can be "mandatory" or "optional"
 |mandatory| Enable mutual SSL. The client should be verified and the mutual SSL handshake must be passed. the API request will be unauthorized if the handshake is failed.|
 |optional|Disable mutual SSL. Even if the mutual SSL handshake failed, the API request is not necessarily unauthenticated. If the mutual SSL handshake is passed, the request will be filtered by the mutual SSL handler.|
 
+
+### Support Mutual SSL Authentication for an API
+
+The Microgateway supports mutual SSL at the API level. It validates the certificate against per API by the name, version, and alias. There are two scenarios in Mutual SSL, the first one is Client directly connects with Microgateway and the second one is the load balancer in front of the Microgateway.
+![Mutual SSL overview]({{base_path}}/assets/img/mutualssl.png)
```

Comment:
https://docs.wso2.com/display/MG320/Mutual+SSL+Authentication?preview=/148931521/148931863/mutualssl.svg%20(1).png need to add this to the PR